duckduckgo/iOS
# Autofill Management View UI changes

Task/Issue URL: https://app.asana.com/0/72649045549333/1202809065048065/f
Tech Design URL:
CC: @THISISDINOSAUR 

**Description**:
Updates tackling all outstanding items except notes. Also includes dark mode support

Note that layout is only focussed on iPhone portrait for now, will tackle landscape + iPad support in another PR 



**Steps to test this PR**:
1. Run through the various UI states in the management view including new, edit and view ensuring everything is in line with designs
2. Update a password in edit mode 
3. Confirm Open Website button is available + works when website URL is a valid URL 
4. Test deleting a login (and undoing it)
5. Also review in dark mode



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
